src/MedicalCard/MedicalCard.jsx

```diff
@@ -5,7 +5,7 @@ import SimpleIcon from "./SimpleIcon";
 import DetailedIcon from "./DetailedIcon";
 import missingAvatar from "../../public/images/empty-avatar.svg";
 import MedicalCardStyles from "./medicalCard.style";
-import DateDisplay from "../dates/DateDisplay";
+import { DateDisplay } from "../index";
```

Comment:
We shouldn't be doing this. It should be ideally ```import DateDisplay from "../dates/``` or the original form. I know the idea is to enforce using the index but we should do this inside the stories, not inside our actual source.

Reply:
I will revert it, but it is the same thing. index.js is being bundled and served and it is already there locally.

Reply:
well in normal source we wouldn't use a concept like this, index.js only exists to serve the library / packaging tools we're using. Its not unfeasible that these tools would change and require, for example, a index.src.js file or something.